E-Commerce/E-Commerce/Presentation/Views/Home/HomeTableView/HomeTableViewCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  HomeTableViewCell.swift
+//  E-Commerce
+//
+//  Created by Orlando Nicolas Marchioli on 15/01/2024.
+//
+
+import UIKit
+
+class HomeTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var itemName: UILabel!
+
+    @IBOutlet weak var itemEliminate: UIButton!
+    @IBOutlet weak var itemImage: UIImageView!
+    @IBOutlet weak var itemType: UILabel!
+    @IBOutlet weak var itemPrice: UILabel!
+    
```

Comment:
I suggest adding the component type to the name to make it easier to understand when you call it later, for example `deleteItemButton` or `itemTypeLabel`